packages/core/core/src/factories/useUserFactory.ts

```diff
@@ -55,41 +64,57 @@ export const useUserFactory = <USER, UPDATE_USER_PARAMS, REGISTER_USER_PARAMS ex
       username: string;
       password: string;
     }) => {
+      Logger.debug('useUserFactory.login', loginUserData);
+
       loading.value = true;
       try {
         user.value = await factoryParams.logIn(loginUserData);
       } catch (err) {
+        Logger.error('useUserFactory.login', err);
+
         throw new Error(err);
       } finally {
         loading.value = false;
       }
     };
 
     const logout = async () => {
+      Logger.debug('useUserFactory.logout');
+
       try {
         await factoryParams.logOut();
         user.value = null;
       } catch (err) {
+        Logger.error('useUserFactory.err');
+
         throw new Error(err);
       }
     };
 
     const changePassword = async (currentPassword: string, newPassword: string) => {
+      Logger.debug('useUserFactory.changePassword', { currentPassword, newPassword });
```

Comment:
should the passwords be logged in full? would it make sense to instead mask them ie show only first and last character. you can tell that the correct value is passed without accidentally logging people passwords